doc/src/devdocs/libgit2.md

```diff
@@ -93,6 +93,7 @@ Base.LibGit2.lookup_branch
 Base.LibGit2.mirror_callback
 Base.LibGit2.mirror_cb
 Base.LibGit2.message
+Base.LibGit2.merge_analysis
```

Comment:
Maybe add the newly documented enums in here too?

Reply:
I don't think we have the other enums in this...

Reply:
Seems like we should though, at least IMO.

Reply:
See [here](https://github.com/JuliaLang/julia/pull/23420#discussion_r135419373) for a discussion of why we didn't do this